style.css

```diff
@@ -0,0 +1,215 @@
+html {
+}
+body { 
+  font-family: 'Helvetica', 'Arial', 'sans-serif';
+  text-align: center;
+  font-size: 1em;
+  line-height: 1.625em;
+}
+div {
+  display: block;
+}
+.container {
```

Comment:
so since this is selecting a class, it may be best to separate it from the HTML native elements like `p, h1, h2` etc. The idea being that you keep your classes/id's separate from the native elements.